lib/urql.ts

```diff
@@ -7,16 +8,17 @@ import { createClient } from '@urql/core';
 const requestPolicy = 'network-only';
 
 export const nftBackedLoansClient = createClient({
```

Comment:
though still not understanding how this subgraph client will be dynamically generated. seems we need to do that at runtime now with our new approach, for both the client and the server

Reply:
yeah this PR was from before we were planning on jumping right into the dynamic stuff. there will need to be another approach here